moduleframework/tests/static/helpmd_lint.py

```diff
@@ -47,31 +47,40 @@ def setUp(self):
     def tearDown(self, *args, **kwargs):
         pass
 
+    def get_msg(self, msg):
+        return msg + " is missing in help.md file."
     def test_helpmd_exists(self):
-        self.assertTrue(self.helpmd)
+        self.assertTrue(self.helpmd, msg="Help.md file is not present")
 
     def test_helpmd_image_name(self):
         container_name = self.dp.get_docker_specific_env("NAME=")
         if container_name:
-            self.assertTrue(self.helpmd.get_image_name(container_name[0].split('=')[1]))
+            self.assertTrue(self.helpmd.get_image_name(container_name[0].split('=')[1]),
+                            msg=self.get_msg("image name"))
 
     def test_helpmd_maintainer_name(self):
         maintainer_name = self.dp.get_specific_label("maintainer")
         if maintainer_name:
-            self.assertTrue(self.helpmd.get_maintainer_name(maintainer_name[0]))
+            self.assertTrue(self.helpmd.get_maintainer_name(maintainer_name[0]),
+                            msg=self.get_msg("maintainer"))
```

Comment:
The error could also be that they don't match, right?

Reply:
You mean like typo?